### PR TITLE
Better wording / order change

### PR DESCRIPTION
**Pull request by Creeperz653**
While i was updating the wording i saw an ordering mistake.

### CODE REVIEW

File: _docs/other/community-geyser-projects.md [hunk 1]
```diff
@@ -25,19 +25,27 @@ A Spigot shop plugin that uses Bedrock forms.
...
 * [GitHub](https://github.com/Camotoy/BedrockSkinUtility)
 
 Creator: [Camotoy](https://github.com/Camotoy)
 
-## Bedrock Player Manager for Floodgate
-A Spigot plugin that is able to run certain commands when a Bedrock player joins, or have Bedrock players able to bypass the anticheat, etc. 
+## CreeperNation Skin API
 
-* [Spigot page](https://www.spigotmc.org/resources/82278/)
+an SkinAPI that displays Java or Bedrock skins based on the UUID or username
```
Comment (Jens-Co) [inline _docs/other/community-geyser-projects.md:44]: Captial. and a dot at end of sentence. :)

Comment (Creeperz653) [reply _docs/other/community-geyser-projects.md]: Done

**Review by Jens-Co — APPROVED** [review]

**Comment by Creeperz653** [comment]
I got the order by sorting from the third letter or second since projects could have the same letter so I moved to the next letter if it was the same I’d move again etc

**Comment by Creeperz653** [comment]
I fixed a link bug with my api

**Comment by Creeperz653** [comment]
That’s what that commit is yeah I am probably lucky this has not been merged yet I got time to finish updating

**Comment by Creeperz653** [comment]
I don't mean to bother about it but the old link doesn't work and I kinda need this to be merged so no one has problems

**Pull request merged**